EnableIPythonDisplay.swift

```diff
@@ -80,3 +80,10 @@ extension PythonObject {
 }
 
 IPythonDisplay.enable()
+
+func display(base64Png: String) {
```

Comment:
Some suggestions on naming conventions:
- `Png` -> `PNG`.
- Compensate for weak type information. Since arguments passed in are not base64 images, it would be good for the argument label to describe role of the argument accurately. How about `encodedBase64PNG:`?

Suggestion:
```swift
func display(encodedBase64PNG encoded: String)
```

Reply:
Or `base64EncodedPNG`. Perhaps this one is better.